test/unit/org/apache/cassandra/db/guardrails/GuardrailAlterTableTest.java

```diff
@@ -56,29 +56,29 @@ public void testAlterTableGuardrail() throws Throwable
         setGuardrail(true);
         assertValid("ALTER TABLE %s WITH compression = { 'class' : 'SnappyCompressor', 'chunk_length_in_kb' : 32 };");
 
-        // Confirm works when disabled
+        // Confirm guardrail works when disabled
```

Comment:
All this negative and double-negative stuff is ripe for confusion. Comment makes it read that the guardrail works, not the statement. Maybe `Confirm ALTER TABLE forbidden when alter_table_enabled is false` is more explicit.

Reply:
I'm all for being more explicit w/this until the other changes go in to create an EnableFlag for us. Good idea.